DOCUMENTATION.md

```diff
@@ -104,751 +29,740 @@ import uk.gov.service.notify.SendSmsResponse;
+#### mobileNumber (required)
 
-If the request is successful, the SendSmsResponse is returned from the client. Attributes of the SendSmsResponse are listed below.
+The phone number of the recipient of the text message. This number can be UK or international.
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+```
+String mobileNumber="+447900900123";
+```
 
-```java
-    UUID notificationId;
-    Optional<String> reference;
-    UUID templateId;
-    int templateVersion;
-    String templateUri;
-    String body;
-    Optional<String> fromNumber;
+#### templateId (required)
 
+The ID of the template. You can find this by signing into [GOV.UK Notify](https://www.notifications.service.gov.uk/) and going to the __Templates__ page.
+
+```
+String templateId="f33517ff-2a88-4f6e-b855-c550268ce08a";
 ```
 
-Otherwise the client will raise a `NotificationClientException`:
+#### personalisation (optional)
 
-|status code|error message|
-|:---|:---|
-|`429`|`429 {`<br>`"errors":`<br>`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type live of 60 requests per second"`<br>`}]`<br>`}`|
-|`429`|`429 {`<br>`"errors":`<br>`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (50) for today"`<br>`}]`<br>`}`|
-|`400`|`400 {`<br>`"errors":`<br>`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient using a team-only API key"`<br>`}]`<br>`}`|
-|`400`|`400 {`<br>`"errors":`<br>`[{`<br>`"error": "BadRequestError",`<br>`"message": ""Can"t send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode""`<br>`}]`<br>`}`|
+If a template has placeholder fields for personalised information such as name or reference number, you must provide their values in a map. For example:
 
-</details>
+```java
+Map<String, String> personalisation = new HashMap<>();
+personalisation.put("first_name", "Amala");
+personalisation.put("application_date", "2018-01-01");
+```
+If a template does not have any placeholder fields for personalised information, you must pass in an empty map or `null`.
 
-#### Arguments
+#### reference (optional)
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+A unique identifier. This reference can identify a single unique notification or a batch of multiple notifications. If you do not have a reference, you must pass in an empty string or `null`.
 
-##### `mobileNumber`
-The mobile number the SMS notification is sent to.
+```
+String reference='STRING';
+```
 
-##### `templateId`
+#### smsSenderId (optional)
 
-The template id is visible on the template page in the application.
+A unique identifier of the sender of the text message notification. To set this up:
 
-##### `reference`
-An optional unique identifier for the notification or an identifier for a batch of notifications. `reference` can be an empty string or null.
+1. Sign into your GOV.UK Notify account.
```

Comment:
I find these instructions to verbose, but they are consistent with the python client. Perhaps we can try to shorten this at another time.

Reply:
It's worth seeing how it goes in research.